magento/magento2
# AC-8286: Creditmemo on non-invoiced lines #37196

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#37196

### Manual testing scenarios (*)

1. Create an order with 2 products, payment by check
2. Create an invoice on 1 product only
3. Start creating a creditmemo
4. Only invoiced line can be refund. The other line is hidden

### Questions or comments


### Contribution checklist (*)
